src/Bridge.ts

```diff
@@ -833,26 +901,40 @@ export class Bridge {
             }
             if (!handled && this.config.checkPermissionAny(event.sender, BridgePermissionLevel.manageConnections)) {
                 // Divert to the setup room code if we didn't match any of these
-                try {
-                    await (
-                        new SetupConnection(
+
+                const botUsersInRoom = this.botUsersManager.getBotUsersInRoom(roomId);
+                // Try each bot in the room until one handles the command
+                for (const botUser of botUsersInRoom) {
+                    try {
+                        const setupConnection = new SetupConnection(
```

Comment:
Could setup connection instead take the `botUsersInRoom` and iterate within the connection, rather than recreating the connection in a loop?

Reply:
I think we decided this was fine in the end.